src/Libraries/RevitNodes/Elements/FamilyInstance.cs

```diff
@@ -155,6 +155,9 @@ private void InitFamilyInstance(Autodesk.Revit.DB.FamilySymbol fs, XYZ pos)
 
         private void InternalSetLevel(Autodesk.Revit.DB.Level level)
         {
+            if (InternalFamilyInstance.LevelId.Compare(level.Id) == 0)
```

Comment:
Yes, I got it but what is the level set to the first time it is created before it is set? The reason I ask is since it looks like it is set in this function so there will be this check even before it is set for the first time, isn't it? I suppose it is some non-null value.